### PR TITLE
Chore: Show Warning for Minimum Duration

### PR DESCRIPTION
**Pull request by RakeshUP**
Task: [APP-1436](https://aragonassociation.atlassian.net/browse/APP-1436)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1436]: https://aragonassociation.atlassian.net/browse/APP-1436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Fabricevladimir** [comment]
Hey @RakeshUP, can you rebase on top of develop, please. It's not playing well at all with my local setup 😢 

Edit: Preview to the rescue!

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 1]
```diff
@@ -375,6 +419,7 @@ const ConfigureCommunity: React.FC = () => {
                   }
                   placeholder={'0'}
                   min="0"
+                  disabled={durationDays === '365'}
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx]: Would be nice to have this in a constant like `MAX_DURATION_DAYS`

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 2]
```diff
@@ -41,13 +77,20 @@ const ConfigureCommunity: React.FC = () => {
...
 
         if (days > 0) {
-          setValue('durationDays', Number(getValues('durationDays')) + days);
+          setValue(
+            'durationDays',
+            (Number(getValues('durationDays')) + days).toString()
+          );
         }
+      } else if (value === 0 && durationDays === '0') {
+        setValue('durationHours', '1');
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx]: Same thing for this so we don't have to hunt it all over the place if a change is needed.

**Review by Fabricevladimir — COMMENTED** [review]
Thanks 🙂, left a couple of minor comments

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 3]
```diff
@@ -413,10 +461,24 @@ const ConfigureCommunity: React.FC = () => {
...
+          />
+        ) : durationDays === '0' &&
+          durationHours === MIN_DURATION_HOURS.toString() &&
+          durationMinutes === '0' ? (
+          <AlertInline
+            label={t('alert.minDurationAlert') as string}
+            mode="warning"
+          />
+        ) : (
+          <AlertInline
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx:491]: Most of this works for me, although I think it is a bit awkward. That said, I encountered an issue when you remove the value from the DAYS field (which shows an appropriate error) and then add 24+ hours to the HOURS field. The behavior is that the DAYS field get updated with the appropriate hours, but the error is not cleared.

image

**Review by Fabricevladimir — CHANGES_REQUESTED** [review]

File: packages/web-app/src/containers/configureCommunity/index.tsx (hunk 3 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/configureCommunity/index.tsx:491]: Made some changes, please check and let me know

Comment (Fabricevladimir) [reply packages/web-app/src/containers/configureCommunity/index.tsx:491]: Looks good 👍🏾

**Review by Fabricevladimir — APPROVED** [review]

**Review by WissenIstNacht — APPROVED** [review]
Looks good, thanks!

**Pull request merged**